pkg/addons/data/DaemonSet.apps/openshift-azure-logging/mdsd.yaml

```diff
@@ -22,10 +22,17 @@ spec:
         livenessProbe:
           exec:
             command:
-            - /bin/bash
-            - /td-agent/liveness-probe.sh
-          failureThreshold: 5
-          initialDelaySeconds: 60
+            - /bin/sh
+            - -c
+            - "LIVENESS_THRESHOLD_SECONDS=${LIVENESS_THRESHOLD_SECONDS:-300}; \nSTUCK_THRESHOLD_SECONDS=${LIVENESS_THRESHOLD_SECONDS:-900};
```

Comment:
why is this better than no liveness probe?

Reply:
from what I read online, fluentd likes just to hangs and stop producing logs... the process is running and nothing is happening...

Reply:
even with http liveness check people say it might hang and still keep reporting status

Reply:
it might - if so, we should fix it.  All the references I saw to that were in 2016/2017 though?